### DISCUSSION TITLE
FN6 Question

### DISCUSSION THREAD

**Discussion by tdegeorge** [TeamBasedInquiryLearning/precalculus#442] [General]
FN6 focuses on inverses and some activities suggest students to "confirm" two functions are inverses by doing composition. But right now, there is no remark or definition about how compositions can help determine if two functions are inverses. Should we add a remark or something somewhere about that? @kathypinzon @AbbyANoble

**Comment by kathypinzon** [comment]
yeah, we need a remark that says something like 
Two functions are inverses if f(g(x))=x......
or maybe we need an activity to discover it.

**Reply by siwelwerd** [reply]
Isn't that [Definition 2.6.4](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/FN6.html#def-inverse-function) ?

**Reply by tdegeorge** [reply]
OMG! How did I miss that??? Ok, nevermind LOL
